src/components/ShowCard.tsx

```diff
@@ -46,15 +46,15 @@ export default function ShowCard(props: MovieCardProps): JSX.Element {
     return (
         <>
```

Comment:
We can remove this fragment now that we have a single container div I believe

Reply:
Current setup doesn't allow it. With a JSX conditional I believe a parent element is required and for now the fragment acts as one.

Reply:
Good point. I am thinking it would make more sense to conditionally render this component, rather than conditionally rendering the content of the card.
```ts
{details && <ShowCard />}
```
I will make a ticket for this change.